src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/PackageLibs.targets

```diff
@@ -140,6 +140,12 @@
       <FilesToPackage Condition="'%(FilesToPackage.SubFolder)' != ''">
         <TargetPath>%(TargetPath)%(FilesToPackage.SubFolder)</TargetPath>
       </FilesToPackage>
+
+      <!-- Include source files for symbol packages. -->
+      <FilesToPackage Include="@(Compile->'%(FullPath)')" Condition="$([System.String]::new('%(FullPath)').StartsWith('$(ProjectDir)'))">
+        <TargetPath>src\$([System.String]::new('%(FullPath)').Replace('$(ProjectDir)', ''))</TargetPath>
```

Comment:
Yeah, I used that for a while but it breaks OSX: https://github.com/Microsoft/msbuild/issues/429. (It also has the risk of including `..\..\..` if there are any code files coming from `ToolRuntime` on TFS, although I changed this before testing TFS so I don't know for sure if the paths lead there.)